tests/test_integration/test_openephys.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from spikewrap.process._loading import load_data
+
+
+def create_dummy_openephys_folder(base_dir):
```

Comment:
I think you have read a bit more on `pytest` and are now familar with fixtures etc. (judging by their use in #205). You could similarly use a fixture here (which takes `tmp_path` and returns `session_path`) then `test_legacy_openephys_detection` takes the `session_path`